fuel-vm/src/interpreter.rs

```diff
@@ -159,6 +152,15 @@ impl<S, Tx> Interpreter<S, Tx> {
     }
 }
 
+#[cfg(feature = "profile-gas")]
+fn current_location(
+    current_contract: Option<ContractId>,
+    pc: crate::constraints::reg_key::Reg<{ crate::constraints::reg_key::PC }>,
```

Comment:
Maybe these should be aliased somehow? This seems quite verbose / not very ergonomic.

Reply:
This is because of feature flags...
There's no nice way to do this but you normally don't need to do it like this when it's not behind a feature flag.